dusty/commands/utils.py

```diff
@@ -20,3 +23,29 @@ def exec_docker(*args):
     updated_env.update(get_docker_env())
     args += (updated_env,)
     os.execle(_executable_path('docker'), 'docker', *args)
+
+def pty_fork(*args):
```

Comment:
It needs the Docker env stuff from the Docker module, if we put it higher we'll get circular imports